core/iwasm/interpreter/wasm_loader.c

```diff
@@ -1386,6 +1386,128 @@ load_memory_import(const uint8 **p_buf, const uint8 *buf_end,
     return false;
 }
 
+#if WASM_ENABLE_TAGS != 0
+static bool
+load_tag_import(const uint8 **p_buf, const uint8 *buf_end,
+                const WASMModule *parent_module, /* this module ! */
+                const char *sub_module_name, const char *tag_name,
+                WASMTagImport *tag, /* structure to fill */
+                char *error_buf, uint32 error_buf_size)
+{
+    _EXCEDEBUG("load_tag_import: %s.%s\n", sub_module_name, tag_name);
+    _EXCEDEBUG(
+        "load_tag_import: parent_module has %d tags total and %d import\n",
+        parent_module->tag_count, parent_module->import_tag_count);
+    _EXCEDEBUG("load_tag_import: parent_module has %d types total\n",
+               parent_module->type_count);
+
+    WASMExport *export = 0;
+    WASMModule *sub_module = NULL;
+
+#if WASM_ENABLE_MULTI_MODULE != 0
+    if (!wasm_runtime_is_built_in_module(sub_module_name)) {
+        sub_module = load_depended_module(parent_module, sub_module_name,
+                                          error_buf, error_buf_size);
+        if (!sub_module) {
+            return false;
+        }
+    }
+
+    // linked_memory = wasm_loader_resolve_memory
+#endif
+
+    WASMModuleCommon *module_reg =
+        wasm_runtime_find_module_registered(sub_module_name);
+    if (!module_reg) {
+        set_error_buf(error_buf, error_buf_size,
+                      "load_tag_import: registered module not found");
+        goto fail;
+    }
+    sub_module = (WASMModule *)module_reg;
+    _EXCEDEBUG(
+        "load_tag_import: sub_module has %d tags total and %d imported tags\n",
+        sub_module->tag_count, sub_module->import_tag_count);
+    _EXCEDEBUG("load_tag_import: sub_module has %d exports total\n",
+               sub_module->export_count);
+    _EXCEDEBUG("load_tag_import: sub_module has %d types total\n",
+               sub_module->type_count);
+
+    export = sub_module->exports;
+    for (uint32 i = 0; i < sub_module->export_count; i++, export ++) {
```

Comment:
Had better declare `uint32 i` in a single line, here it is C source file, we usually keep the coding style of C.

Reply:
Updated code according to coding style